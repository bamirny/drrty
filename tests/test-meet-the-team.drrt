def int main() {
	list[str] team;
	int i;
	list[str] job;
	list[str] img;
	
	team = ["Dylan","Richard","Rania","Trinity"];


	job = ["Manager","Language Guru", "Tester", "System Architect"];

	img = ["https://cdn.pixabay.com/photo/2021/12/16/20/13/ducks-6875155__340.jpg","https://cdn.pixabay.com/photo/2021/10/08/06/45/seagulls-6690361__340.jpg","https://cdn.pixabay.com/photo/2021/11/06/22/05/camels-6774540__340.jpg","https://cdn.pixabay.com/photo/2021/11/16/08/01/animal-6800387__340.jpg"];

	makeHeader("Meet the DRRTY Team!");

	
	for(i=0; i<team.length();i = i+1){

		makeText(team.get(i));
		createElement("h2",job.get(i),"");

		makeImage(img.get(i));
	}
	return 0;

}