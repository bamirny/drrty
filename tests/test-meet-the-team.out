<h1>Meet the DRRTY Team!</h1>
<h2>Dylan</h2>
<li>Manager</li>
<img src='https://cdn.pixabay.com/photo/2021/12/16/20/13/ducks-6875155__340.jpg'>
<h2>Richard</h2>
<li>Language Guru</li>
<img src='https://cdn.pixabay.com/photo/2021/10/08/06/45/seagulls-6690361__340.jpg'>
<h2>Rania</h2>
<li>Tester</li>
<img src='https://cdn.pixabay.com/photo/2021/11/06/22/05/camels-6774540__340.jpg'>
<h2>Trinity</h2>
<li>System Architect</li>
<img src='https://cdn.pixabay.com/photo/2021/11/16/08/01/animal-6800387__340.jpg'>
